data-entry/invoice-editor/invoice-editor-flow/src/main/java/com/vaadin/invoice/editor/Invoice.java

```diff
@@ -31,15 +30,7 @@ public Invoice(String description, int price, Currency currency, int vat, int am
 
     @Override
     public String toString() {
-        return "Invoice with id " + id;
-    }
-
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        this.id = id;
+        return " product " + product;
```

Comment:
This doesn't make sense as the string representation of one invoice. It's now hardcoded to work only for that one use case, but the " product " should be part of the notification logic. Maybe return just product name?

Reply:
Updated to return just `product`.